physics/closed_system.hpp

```diff
@@ -24,42 +25,51 @@ using quantities::MomentOfInertia;
 // Effectively an extension of
 // |BarycentreCalculator<DegreesOfFreedom<InertialFrame>, Mass>| that also
 // handles rotational motion.
-// Note: As a consequence of our separate handling of vectors and bivectors, the
-// inertia tensor used here does *not* follow the convention customary in
-// physics.  The usual convention is
-//  ⎛ ∑(y² + z²)   -∑xy      -∑xy    ⎞
-//  ⎜    -∑yx   ∑(x² + z²)   -∑yz    ⎟
-//  ⎝    -∑zx      -∑zy   ∑(x² + y²) ⎠
-// but our convention is:
-//  ⎛ ∑x²   ∑xy   ∑xy ⎞
-//  ⎜ ∑yx   ∑y²   ∑yz ⎟
-//  ⎝ ∑zx   ∑zy   ∑z² ⎠
-// This leads to simpler transformation formulæ.  Where the physics usage would
-// be to multiply the inertia tensor with a pseudovector, we must instead take
-// the anticommutator with the bivector: L = Iω becomes L = {I, ω}, and the
-// moment of inertia αIα along an axis α becomes ⟨α, {I, α}⟩.
+// |InertialFrame| is an inertial frame in which the motion of the member bodies
+// is given; |SystemFrame| is a non-rotating frame with the same axes as
+// |InertialFrame| and whose origin is the centre of mass of |SystemFrame|.
+//
+// Note: It is more natural to manipulate the inertia tensor as a
+// |SymmetricBilinearForm<MomentOfInertia, Frame, Vector>| than a
+// |SymmetricBilinearForm<MomentOfInertia, Frame, Bivector>|.
+// The former has coordinates
+//   ⎛ ∑x²   ∑xy   ∑xy ⎞
+//   ⎜ ∑yx   ∑y²   ∑yz ⎟
+//   ⎝ ∑zx   ∑zy   ∑z² ⎠
+// whereas the latter (obtained from the former by |Anticommutator()| has the
+// coordinates
+//   ⎛ ∑(y² + z²)   -∑xy      -∑xy    ⎞
+//   ⎜    -∑yx   ∑(x² + z²)   -∑yz    ⎟
+//   ⎝    -∑zx      -∑zy   ∑(x² + y²) ⎠
+// It is however common practice in physics to use the bilinear form on
+// bivectors, and to call its eigenvalues the principal moments of inertia. When
+// interfacing with systems that use those, |AnticommutatorInverse()| and
+// |Anticommutator()| should be used.
```

Comment:
I am wondering if it's wise to expose have the API of that class (as opposed to its internals) use the form on `Vector`.  It will require that all the clients call `Anticommutator` and `AnticommutatorInverse` at the right places, including at places where there is no strong typing because we are interfacing with KSP.  This seems bug-prone.  Maybe this class should have an API using the form on `Bivector` (it's in directory `physics`, after all) and call  `Anticommutator` and `AnticommutatorInverse` internally.